src/modules/batch-calculator/batch-calculator.service.ts

```diff
@@ -20,10 +20,13 @@ export class BatchCalculatorService {
      * This runs concurrently to rank all the posts
      *
      * @param batch
-     * @param minScore
+     * @param rejectScore
      */
-    async batchCalculate(batch: RawPost[], minScore: number): Promise<SortedPost[]> {
-        strictEqual(minScore >= 0, true, 'batchCalculate -> minScore must be >= 0');
+    async batchCalculate(batch: RawPost[], rejectScore: number): Promise<SortedPost[]> {
+        strictEqual(rejectScore >= 0, true, 'batchCalculate -> minScore must be >= 0');
```

Comment:
this variable was updated to be named rejectScore however the associated assert message was not

```suggestion
        strictEqual(rejectScore >= 0, true, 'batchCalculate -> rejectScore must be >= 0');
```